qiskit/extensions/quantum_initializer/ucrx.py

```diff
@@ -89,7 +89,17 @@ def ucx(self, angle_list, q_controls, q_target):
     if num_contr != len(q_controls):
         raise QiskitError("Number of controlled rotations does not correspond to the number of"
                           " control-qubits.")
-    return self.append(UCX(angle_list), [q_target] + q_controls, [])
+    return self.append(UCRXGate(angle_list), [q_target] + q_controls, [])
+
+
+def ucx(self, angle_list, q_controls, q_target):
+    """
+    Deprecated version of ucrx.
+    """
+    import warnings
+    warnings.warn('qc.ucx is deprecated, use qc.ucrx instead!', DeprecationWarning, 2)
```

Comment:
This message does not match the style of other deprecation messages.
```suggestion
    warnings.warn('The QuantumCircuit. ucx() method is deprecated as of 0.14.0, and '
                  'will be removed no earlier than 3 months after that release date. '
                  'You should use the QuantumCircuit. ucrx() method instead.',
                  DeprecationWarning, stacklevel=2)
```